crates/e2e-testing/src/spin.rs

```diff
@@ -113,12 +109,13 @@ pub fn registry_login(registry_url: &str, username: &str, password: &str) -> Res
         ],
         None,
         None,
-    )
+    )?;
+    utils::assert_success(&output);
+    Ok(output)
 }
 
 pub fn version() -> Result<String> {
-    match utils::run(vec!["spin", "--version"], None, None) {
-        Ok(output) => Ok(format!("{:#?}", std::str::from_utf8(&output.stdout)?)),
-        Err(err) => Err(err),
-    }
+    let output = utils::run(&["spin", "--version"], None, None)?;
+    utils::assert_success(&output);
```

Comment:
We may eventually remove these assertions in favor of the top-level testing code asserting success. For now though we'll leave this here as every test assumes that these calls succeed.

Reply:
👍